test/MUXControls.Test/Infra/TestHelpers.cs

```diff
@@ -231,6 +252,7 @@ public virtual void Dispose()
     public class TestCleanupHelper
     {
         public static int TestSetupHelperPendingDisposals { get; set; }
+        public static int TestPagePendingDisposals { get; set; }
```

Comment:
This shouldn't need to be static. It should just be a private instance member on TestSetupHelper. Right?